api/app/db/models/common.py

```diff
@@ -13,6 +13,7 @@ class TZTimeStamp(TypeDecorator):  # pylint: disable=abstract-method
     """ TimeStamp type that ensures that timezones are always specified.
     If the timezone isn't specified, you aren't guaranteed that you're going to get consistent times. """
     impl = TIMESTAMP(timezone=True)
+    cache_ok = True
```

Comment:
Is this a sqlalchemy thing?

Reply:
Yeah - it's a new warning (sqlalchemy upgrade?) that we started getting. Something about it not knowing if it can use this column to cache things.